guide/lib/examples/password.rb

```diff
@@ -0,0 +1,31 @@
+module Examples
+  module Password
+    def default_password
+      <<~PASSWORD
+        = f.govuk_password_field :password, label: { text: "Enter your password" }
+      PASSWORD
+    end
+
+    def password_with_a_captioned_heading_label_and_hint
+      <<~PASSWORD
+        = f.govuk_password_field(:pin,
+            label: { text: "Enter your PIN", tag: "h2", size: "l" },
+            caption: { text: "Security", size: "m" },
+            hint: { text: "Your PIN was emailed to you when you registered for this service" })
```

Comment:
Should probably use another example, only referring to passwords, as the Design System guidance states that this input should only be used for passwords, and not other types of security information.

Reply:
Fair - I need unique attribute names here which is why I used PIN. I'll rename to `password_1`, `password_2` and `password_3`.